### ISSUE TITLE
This project will be continued in another place?

### ISSUE THREAD

**Issue by lautarobock** [justindujardin/ng2-material#333]
Hi, I've noticed that you deprecate this project and down the gh-page. 
Do you think to continue it in another place?
Is there a way to upload gh-page again?
I found a couple of components very useful!

**Comment by almothafar** [comment]
Look at this: https://github.com/angular/material2

**Comment by lautarobock** [comment]
Yes, I know that. But there is some components missing like data-table.

**Comment by ollwenjones** [comment]
@lautarobock this was a community effort started before material2 was in alpha.0, but contributions have moved from this to that, and this is officially 'deprecated' - so you can use it, but there won't ongoing contributions here, so you're on your own for any bug-fixes or improvements, etc.

The official Material 2 data table [is in the works](https://github.com/angular/material2/issues/581), otherwise there are some good data-table components out there you can style to look Material.

The data-table in here was mostly SCSS anyway, (with a selection model on top of an `ngFor`) which you're welcome to lift,

**Comment by ollwenjones** [comment]
https://www.npmjs.com/package/angular2-data-table is what we were about to switch to, before I got moved onto a React project.

**Comment by lautarobock** [comment]
Thank you very much!
